File: backend/components/metiers/threed/legacy/documentation/models/technique/model_generator.py
"""
Script Python pour générer un modèle de documentation legacy Threed.
Ultra avancé, conforme au cahier des charges.
"""


def generate_model(name, fields):
    print(f"# Modèle : {name}\n")
    for field in fields:
        print(f"- {field}")


if __name__ == "__main__":
    name = "Documentation technique"
    fields = [
        "Titre",
        "Résumé",
        "Architecture",
        "API",
        "Sécurité",
        "Déploiement",
        "Maintenance",
    ]
    generate_model(name, fields)

File: backend/components/metiers/threed/legacy/documentation/models/technique/model_generator.js
// Script JS pour générer un modèle de documentation legacy Threed
// Ultra avancé, conforme au cahier des charges
function generateModel(name, fields) {
  console.log(`# Modèle : ${name}\n`);
  fields.forEach(field => console.log(`- ${field}`));
}

if (require.main === module) {
  const name = 'Documentation technique';
  const fields = ['Titre', 'Résumé', 'Architecture', 'API', 'Sécurité', 'Déploiement', 'Maintenance'];
  generateModel(name, fields);
}
